manifests/p/PTRTECH/UVtools/3.1.1/PTRTECH.UVtools.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PTRTECH.UVtools
+PackageVersion: 3.1.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/sn4k3/UVtools/releases/download/v3.1.1/UVtools_win-x64_v3.1.1.msi
+  InstallerSha256: 31E79FE75616CE084A81D6776C9CB29E869A9C4027E98E6DDF2B2120451D7D41
+  ProductCode: '{9E670331-F239-40B9-A935-52EBA14E9779}'

```

I get a different SHA and ProductCode when I attempt to install it? 
[WinGet-2022-03-21-00-05-05.697.log](https://github.com/microsoft/winget-pkgs/files/8313251/WinGet-2022-03-21-00-05-05.697.log)

```suggestion
  InstallerSha256: BE2ABE5A5585B79967A4D7D6F1D94C4FB20786FA60B3018FBA1481E189F7DCFC
  ProductCode: '{60411DF6-CF1B-4102-A31C-D3BC2A19248D}'
```

---

I've updated the MSI with a minor fix in meanwhile and forget about this. Thanks